Sync-Introduction.md

```diff
@@ -1,23 +1,17 @@
 ---
 uid: Sync-Introduction
-title: Sync Introduction
+title: Downloads & Sync
 legacyUrl: /support/solutions/articles/44001161893-sync-introduction
 ---
 
-With Emby Sync, you can choose specific media, then a device or destination such as a cloud provider along with criteria such as resolution and bit rate and Emby will prepare/convert the media to the format requested, then copy it to the destination.
+The Downloads and Sync feature is an [Emby Premiere](Emby-Premiere.md) feature offering users and server admin, the ability to copy media at specific quality resolution, to devices for offline streaming or to folders to provide backup and converted versions of the media. The latter requires the installation of the **Folder Sync** plugin.
 
-There are a few different types of sync built into Emby. Some of these sync types are only available to the administrator and other can be used by users with correct permissions.
+See [Download Options](Sync.md) for full details.
 
-As an example, an administrator can setup Dropbox or Google Drive as a sync destination, then setup specific media to be synced to the "cloud". In this way the administrator can SYNC popular or new movies/shows to the cloud. When a remote user plays back this media it can be served from the Cloud vs the Emby server saving the server's upload bandwidth. In this scenario the media is prepared and uploaded ONE TIME to the cloud where it can then be played back numerous times.
+Depending on user access rights and the media type, different options for download are presented within the emby client apps to the users to either download media to the device or to another device or to a target folder. You can choose a specific media file, a complete TV Show or a season or even a whole library for download or copy to a predefined **Folder Sync** folder. Options are provided to set the required quality and Emby will prepare/convert the media to the format requested, then copy it to the destination.
```

Comment:
requested, then copy it to the destination.

=> requested, before copying it to the destination.

Reply:
Done  - will upload shortly